DRAFT_RELEASE_NOTES.md

```diff
@@ -31,11 +32,15 @@ To reduce burden of upgrading aiSSEMBLE, the Baton project is used to automate t
 | upgrade-v1-chart-files-aissemble-version-migration | Updates the docker image tags within your project's deployment resources (`<YOUR_PROJECT>-deploy/src/main/resources/apps/`) to use the latest version of the aiSSEMBLE       |
-| spark-version-upgrade-migration                    | Updates the Spark Application executor failure parameters to their new key name to ensure compatibility with spark `3.5`                                                     |
+| spark-version-upgrade-migration                    | Updates the Spark Application executor failure parameters to their new key name to ensure compatibility with Apache Spark `3.5`                                                     |
 | spark-pipeline-messaging-pom-migration             | Updates a Spark pipeline module `pom.xml` with the new CDI classes dependency to ensure messaging compatibility with Java 17                                                             |
 | spark-pipeline-messaging-cdi-factory-migration     | Updates a Spark pipeline module `CdiContainerFactory.java` with the new CDI classes to ensure messaging compatibility with Java 17                                                   |
+| spark-pipeline-servlet-api-migration               | Updates a Spark pipeline module `pom.xml` with the `javax.servlet-api` dependency to ensure compatibility with Apache Spark `3.5` which has not migrated to Jakarta packages yet      |
 | it-infrastructure-java-upgrade-migration           | Updates the Java docker image version in the integration test docker module to JDK 17                                                                                        |
-| log4j-maven-shade-plugin-migration                 | Updates the Maven Shade Plugin with the new Log4j dependency information                                                                                                     |
+| log4j-maven-shade-plugin-migration                 | Updates the Maven Shade Plugin with the new Log4j dependency information                                                                             |
+| quarkus-bom-migration                              | Updates all references to the `quarkus-bom` and `quarkus-universe-bom` to use the new `aissemble-quarkus-bom` for managing Quarkus dependencies                               |
+| pom-dependency-version-migration                   | Updates the pom dependencies previously managed by the aiSSEMBLE `bom-component` to include their necessary versions                                                                |
+| java-package-migration                             | Updates the affected java classes from their old package name to their new package name to ensure compatibility with the updated Java 17 dependencies                              |
```

Comment:
Q: Should we say either here or in the block at the top that this is largely Javax -> Jakarta?

Reply:
Thinking we put it as part of the migration description, feels sort of out of place at the high level block

Reply:
Added to migration description